### ISSUE TITLE
[Bug Report] Snackbar accessibility improvements

### ISSUE THREAD

**Issue by WilcoFiers** [vuetifyjs/vuetify#10047]
### Environment
**Vuetify Version:** 2.1.14
**Vue Version:** 2.6.11
**Browsers:** Chrome 79.0.3945.88
**OS:** Mac OS 10.15.1

### Steps to reproduce
All variations in the snackbar playground have these issues, see https://vuetifyjs.com/en/components/snackbars#snackbars

### Expected Behavior
- Buttons placed in the snackbar, (with or without "text") have a visible focus style
- The appearance of the snackbar is announced by screen readers
- The text on any button placed inside the snackbar is still visible without the "text" attribute set on the button

### Actual Behavior
- v-btn inside a v-snackbar has no visible focus
- the appearance of a v-snackbar is not announced by screen readers
- When "text" is not used on a v-btn inside a v-snackbar, the text color in the light theme is nearly identical to the background color of the button

### Reproduction Link
<a href="https://codepen.io/wilcofiers/pen/gObRyyV?&editable=true&editors=101" target="_blank">https://codepen.io/wilcofiers/pen/gObRyyV?&editable=true&editors=101</a>

### Other comments
The announcement issue should be easy to resolve by putting role="alert" on the snackbar. I looked into the focus indicator issue, but I couldn't quite figure out where that was coming from.

**Comment by MajesticPotatoe** [comment]
@WilcoFiers
> v-btn inside a v-snackbar has no visible focus

Can you elaborate a little on this? are you referring to there being no hover effect like in a standard button? If so would consider breaking this into a separate issue, though may be intended as it existed pre 2.x and may be MD spec related.

> the appearance of a v-snackbar is not announced by screen readers

Should be easily fixable with `role="alert"` as you have mentioned

> when "text" is not used on a v-btn inside a v-snackbar, the text color in the light theme is nearly identical to the background color of the button

Would suggest making this a separate issue for this. Not using text on buttons in a snackbar would be against MD Spec. Seems likely a lack of css definition, but also has been like this pre 2.x.

**Comment by phiter** [comment]
As @MajesticPotatoe said, snackbars should not use non-text buttons, as per the [MD2 spec](https://material.io/components/snackbars/#anatomy), but we can disable the styling of filled buttons.

**Comment by vickyrare** [comment]
@phiter Could you please confirm that v-snackbar a11y and it is announced by the screen readers?
